src/expr/datatype.h

```diff
@@ -763,24 +847,191 @@ class CVC4_PUBLIC Datatype {
    */
   Expr getConstructor(std::string name) const;
 
-  /** get sygus type */
+  /** get sygus type
+   * This gets the built-in type associated with
+   * this sygus datatype, i.e. the type of the
+   * term that this sygus datatype encodes.
+   */
   Type getSygusType() const;
-  /** get sygus var list */
+
+  /** get sygus var list
+   * This gets the variable list of the function
+   * to synthesize using this sygus datatype.
+   * For example, if we are synthesizing a binary
+   * function f where solutions are of the form:
+   *   f = (\ xy. t[x,y])
```

Comment:
Is the '\\' here a typo? Missing quantifier?

Reply:
This is a lambda, however, I realize I'm not using the notation consistently, I'll switch to "lambda xy"